data/hai outfits.txt

```diff
@@ -107,58 +107,67 @@ effect "ion impact"
 
 outfit "Railgun Slug"
 	category "Ammunition"
-	cost 60
+	cost 13
 	thumbnail "outfit/railslug"
-	"mass" 0.02
+	"mass" 0.001
 	"railgun slug capacity" -1
-	description "These railgun slugs incorporate energy-discharging capacitors that deal a small amount of hull damage even when the target's shields are functioning, and they can punch crippling holes in the hull of a ship whose shields are down."
+	description "These railgun slugs incorporate energy-discharging capacitors that deal a small amount of hull damage even when the target's shields are functioning.  The Hai use them in Railguns that propel the slug to relativistic speed, simultaneously using the stored charge to amplify recoil, propelling their Flea drones away from danger, and turning them into effective long range support."
+
+outfit "Railgun Slug Rack"
+	category "Ammunition"
+	cost 15000
+	thumbnail "outfit/railslug_rack"
+	"mass" 1
+	"outfit space" -3
+	"railgun slug capacity" 1500
+	ammo "Railgun Slug"
+	description "The Railgun Slug Rack is used to store and load extra Railgun ammunition."
 
 outfit "Railgun"
 	category "Secondary Weapons"
 	cost 20000
 	thumbnail "outfit/railgun"
-	"mass" 4
-	"outfit space" -7
-	"weapon capacity" -7
+	"mass" 1
+	"outfit space" -4
+	"weapon capacity" -4
 	"gun ports" -1
-	"railgun slug capacity" 150
+	"railgun slug capacity" 3210
 	weapon
 		sprite "projectile/rail slug"
 			"frame rate" 30
 		ammo "Railgun Slug"
 		icon "icon/rail gun"
 		sound "ion impact"
-		"fire effect" "rail sparks" 2
-		"live effect" "rail sparks" 5
+		"fire effect" "rail sparks"
 		"hit effect" "bullet impact"
 		"inaccuracy" 3
-		"velocity" 22
+		"velocity" 44
 		"lifetime" 1
-		"reload" 120
-		"firing energy" 92
-		"firing heat" 39
-		"hit force" 44
+		"reload" 3
+		"firing energy" 1.5
+		"firing heat" 1.8
+		"hit force" 10
+		"firing force" 20
 		"stream"
 		"submunition" "rslug" 1
-	description "The Hai created this compact weapon for their Flea drones, providing mobile long-range support for the close quarters Pond Strider carrier."
+	description "The Hai created this compact, reverse-thrusting weapon for their Flea drones.  Their drone pilots joke that they provide more thrust than engines of similar size, so Unfettered should be using them at the back of their ships.  There's probably a cultural component to the joke that many miss, because it's told quite often and the Hai laugh hysterically about it - every time."
```

Comment:
Perhaps instead of using the term "reverse-thrusting" to describe the Railgun, instead describe it as what it actually is; a high firing force that emulates reverse thrust, providing light ships a tactical advantage.
```suggestion
	description "The Hai created this compact, reverse-thrusting weapon for their Flea drones. Their drone pilots joke that they provide more thrust than engines of similar size, so Unfettered should be using them at the back of their ships.  There's probably a cultural component to the joke that many miss, because it's told quite often and the Hai laugh hysterically about it - every time."
```

Reply:
i'm going to go with "tactical recoil"